src/App.tsx

```diff
@@ -20,6 +24,11 @@ const queryClient = new QueryClient({
       staleTime: 300000,
     },
   },
+  queryCache: new QueryCache({
+    onError: (error) => {
+      console.log('Got error ' + error.message);
```

Comment:
Should probably add a TODO here to keep track that we need to properly implement this